core/http/server-spring/src/main/java/org/eclipse/rdf4j/http/server/repository/RepositoryController.java

```diff
@@ -390,4 +408,18 @@ else if (qr instanceof GraphQueryResult) {
 			return qr;
 		}
 	}
+
+	private static <T> CloseableIteration<T, QueryEvaluationException> paginated(QueryResult<T> qr,
```

Comment:
This return type is a problem, as the ModelAndView renderers expect the object to be of type GraphQueryResult or TupleQueryResult. Will fix.